internal/generator/vector/output/loki/loki_conf_test.go

```diff
@@ -12,8 +12,7 @@ import (
 	"github.com/openshift/cluster-logging-operator/internal/utils"
 	testhelpers "github.com/openshift/cluster-logging-operator/test/helpers"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/ginkgo/extensions/table"
+	"github.com/onsi/ginkgo/v2"
```

Comment:
leave the dot reference, so that the additional `gingko.` additions are not necessary below

Reply:
actually it has package `ginkgo` https://github.com/onsi/ginkgo/blob/master/core_dsl.go#L16
@jcantrill